x-pack/plugins/observability/public/pages/rule_details/components/page_title.tsx

```diff
@@ -35,20 +41,38 @@ export function PageTitle({ rule }: PageHeaderProps) {
       <EuiFlexGroup direction="column" alignItems="flexStart">
         <EuiFlexItem component="span" grow={false}>
           <EuiText color="subdued" size="xs">
-            <b>{LAST_UPDATED_MESSAGE}</b> {BY_WORD} {rule.updatedBy} {ON_WORD}&nbsp;
+            <strong>
+              {i18n.translate('xpack.observability.ruleDetails.lastUpdatedMessage', {
+                defaultMessage: 'Last updated',
+              })}
+            </strong>
+            &nbsp;
+            {BY_WORD} {rule.updatedBy} {ON_WORD}&nbsp;
             {moment(rule.updatedAt).format('ll')} &emsp;
-            <b>{CREATED_WORD}</b> {BY_WORD} {rule.createdBy} {ON_WORD}&nbsp;
+            <strong>
+              {i18n.translate('xpack.observability.ruleDetails.createdWord', {
+                defaultMessage: 'Created',
+              })}
+            </strong>
+            &nbsp;
+            {BY_WORD} {rule.createdBy} {ON_WORD}&nbsp;
```

Comment:
🍰  nit: Not the purpose of this PR but having a translated sentence (last updated at by <author> on <date>) taking the two variables would be better to handle languages using a different grammatical construction of sentences.